lib/util/string_util.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/foundation.dart';
+
+extension StringExt on String {
+  String decode() {
```

Comment:
Decode could be used by a lot of decoding algorithms. So i think this method is too specific for a generic concept.
We can use existing class `ContactFormatters`